beacon-chain/operations/slashings/service.go

```diff
@@ -26,8 +25,9 @@ func NewPool() *Pool {
 }
 
 // PendingAttesterSlashings returns attester slashings that are able to be included into a block.
-// This method will not return more than the block enforced MaxAttesterSlashings.
-func (p *Pool) PendingAttesterSlashings(ctx context.Context, state *beaconstate.BeaconState) []*ethpb.AttesterSlashing {
+// This method will return all pending attester slashings unless the `block` parameter is set to true
+// to indicate the request is for a block.
+func (p *Pool) PendingAttesterSlashings(ctx context.Context, state *beaconstate.BeaconState, block bool) []*ethpb.AttesterSlashing {
```

Comment:
block bool is ambiguous, it sounds like you want the function to block. Perhaps we can call for `forProposal`? or `forBlockProposal`?

Reply:
Maybe we can call it `noLimit` ?